presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
```

Comment:
It would be helpful for an administrator to know what the accuracy problem was.

Reply:
@arhimondr

Reply:
The result was totally incorrect for some rare cases in division. The overflow of an integer was happening, and instead of an unsigned division - signed division was used.

Reply:
Referred to the PR